src/routes/+layout.svelte

```diff
@@ -13,19 +13,16 @@
 		['/settings', 'Settings']
 	];
 
-	let theme = 'light';
+	$: theme = $settingsStore?.userTheme;
 
 	function toggleTheme() {
 		theme = theme === 'light' ? 'dark' : 'light';
 		document.documentElement.setAttribute('data-color-theme', theme);
+		settingsStore.update((store) => {
+			store!.userTheme = theme;
+			return store;
+		});
 	}
-
-	onMount(() => {
-		// Set initial theme based on user's browser preference
-		const prefersDark = window.matchMedia('(prefers-color-scheme: dark)').matches;
-		theme = prefersDark ? 'dark' : 'light';
-		document.documentElement.setAttribute('data-color-theme', theme);
-	});
```

Comment:
I'm not sure if you finished working on this PR but this `prefersDark` was needed to determine what the default color theme should have been.

My browser is set to _"prefers light mode"_, but in [the latest version of this PR](https://f17e6891.hollama.pages.dev/) the default option is to "switch to light mode":

![image](https://github.com/user-attachments/assets/b77d545b-2683-4b80-83eb-2d0474fb064f)

Reply:
> I'm not sure if you finished working on this PR but this `prefersDark` was needed to determine what the default color theme should have been.

I moved this logic to `app.html`, I thought it should have worked the same way as before.


> My browser is set to _"prefers light mode"_, but in [the latest version of this PR](https://f17e6891.hollama.pages.dev/) the default option is to "switch to light mode":

That's weird. It behaves correctly when I try it. I'll look further into that.

Reply:
> That's weird. It behaves correctly when I try it. I'll look further into that.

I'm able to consistently replicate the behavior when I delete the existing `localsStorage` `hollama-settings` and refresh the page.

Reply:
Also, not sure if you know this but you can emulate the browser settings in Inspector with the "Run command" menu:

![image](https://github.com/user-attachments/assets/8d1c00b5-697d-4f3e-b39d-6120534fcbe4)

Reply:
Thanks for the advice!

The problem should be fixed now.